test-network-function/accesscontrol/suite.go

```diff
@@ -54,6 +54,12 @@ var (
 	}
 )
 
+// failedPod is a helper type to track pods that fail in each of the TCs.
+type failedPod struct {
```

Comment:
why not storing pods directly ? then extracting  name and namespace later when needed ?

Reply:
You're right. Done.